lib/status.js

```diff
@@ -33,13 +33,23 @@ class Status {
      * The number of times this circuit breaker has timed out
      */
     this.timeouts = 0;
+    /**
+     * The number of times the cache is used
+     */
+    this.cacheHit = 0;
```

Comment:
Even though I'm not very fond of the naming scheme I originally chose, I think we should be consistent and make this and `cacheMiss` plural: `cacheHits` and `cacheMisses`.